lib/StripeResource.js

```diff
@@ -140,9 +140,7 @@ StripeResource.prototype = {
           if (response.error) {
             var err;
 
-            // These are now available on the top-level resource's
```

Comment:
I removed this comment because on a second read I realized that it's not accurate :/ `response.error` is all that we sent into error initialization, so these properties must be on there.